react/index.js

```diff
@@ -1,28 +1,32 @@
-import React, { Component, Fragment } from 'react'
-import { Input, Button, Badge } from 'vtex.styleguide'
+import React, { Component } from 'react'
+
 import { injectIntl, intlShape } from 'react-intl'
 
+import LoginAsCustomer from './components/LoginAsCustomer'
+import LogoutCustomerSession from './components/LogoutCustomerSession'
+import TelemarketingIcon from './icons/TelemarketingIcon'
 import { setCookie, deleteCookie } from './utils/cookies'
 import { request } from './utils/request'
 import { translate } from './utils/translate'
+import { truncateString } from './utils/format-string'
 import './global.css'
 
 const IMPERSONATED_CUSTOMER_EMAIL = 'vtex-impersonated-customer-email'
 
-/** Canonical Impersonate component */
-class ImpersonateCustomer extends Component {
+/** Canonical Telemarketing component that receives an client email and impersonates that client if the attendant has the correct permission for it. */
```

Comment:
The Canonical Telemarketing component impersonates an attendant, with the right permissions, as a client.

Reply:
👍